### ISSUE TITLE
No interface at all, only text when I visit the location in my web browser

### ISSUE THREAD

**Issue by zanium** [AHAAAAAAA/PokemonGo-Map#2544]
## Current Behavior

Only text when I go to "localhost:5000" no pictures or anything, just text saying "pokemon go map"

## Your Environment

Chrome, latest update

**Comment by pedbsktbll** [comment]
same issue here... I saw this before when I forwarded over https using apache... it just displayed this ugly, broken UI.. when I switched to using nginx to forward over https, then everything worked fine... After pulling develop though, it appears to happen all the time now

**Comment by DeviantEng** [comment]
Yup, same here.  

I ran `pip install -r requirements.txt` and `npm install`, but no dice.  Not seeing a `./static/dist` directory which would contain the css, IIRC.

**Comment by vihangm** [comment]
See #2384
Run the following:
```
npm install
npm install grunt-cli
./node_modules/.bin/grunt build
```
And then you should be good to go.

**Comment by adbspurs** [comment]
Temporary fix I found for Windows: Install master zip download then install develop zip download over the top. Does cause Stats window to not work thought? (I guess that implies a CSS sheet missing in the download?)

**Comment by DeviantEng** [comment]
@vihangm  Yup, that was it.  The old wiki needs to be updated, or the new wiki needs install instructions.

Thanks!

**Comment by zanium** [comment]
@vihangm 

when I went to use the "./node_modules/.bin/grunt build" command, I got 

"Loading "jshint.js" tasks...ERROR
>> Error: Cannot find module 'entities/lib/decode_codepoint.js'
Loading "sass.js" tasks...ERROR
>> Error: Cannot find module 'lodash.clonedeep'
Loading "babel.js" tasks...ERROR
>> Error: Cannot find module 'babel-runtime/helpers/typeof'
Warning: Task "babel" not found. Use --force to continue.

Aborted due to warnings.
"

**Comment by pedbsktbll** [comment]
Hmm I'm having an issue with the "npm install" command.. Maybe some kind of versioning issue?

> 20908 verbose true,/opt/PokemonGo-Map/node_modules,/opt/PokemonGo-Map/node_modules unbuild grunt-sass@1.2.0
20909 info postuninstall grunt-sass@1.2.0
20910 error node-sass@3.8.0 install: `node scripts/install.js`
20910 error Exit status 127
20911 error Failed at the node-sass@3.8.0 install script.
20911 error This is most likely a problem with the node-sass package,
20911 error not with npm itself.
20911 error Tell the author that this fails on your system:
20911 error     node scripts/install.js
20911 error You can get their info via:
20911 error     npm owner ls node-sass
20911 error There is likely additional logging output above.
20912 error System Linux 3.16.0-4-amd64
20913 error command "/usr/bin/nodejs" "/usr/bin/npm" "install"
20914 error cwd /opt/PokemonGo-Map
20915 error node -v v0.10.29
20916 error npm -v 1.4.21
20917 error code ELIFECYCLE
20918 verbose exit [ 1, true ]

**Comment by vihangm** [comment]
@pedbsktbll I'm using node `v0.12` and npm `v2.15` locally and things work with that. And my heroku test deploy was on node `v5.11` and npm `v3.8.6` and that worked too.

You seem to be using node `v0.10.29` and npm `v1.4.21` which might be too old for this project's dependencies.

@zanium don't know what the problem you ran into is. :(

**Comment by vihangm** [comment]
@zanium You could switch to the https://github.com/AHAAAAAAA/PokemonGo-Map/tree/master branch if you want to get things working. Or checkout to a commit before the generated files got removed from the repo. AHAAAAAAA/PokemonGo-Map@0880e523 should be a good enough commit

**Comment by pedbsktbll** [comment]
Thanks... Finally got the npm commands working and had to symlink nodejs for the grunt build, but now I'm getting the same error as @zanium ....

Might just revert that last commit and work from there for now... It claims it aborted due to "warnings", and you can use "--force" to continue... so may just try that too

**Comment by zanium** [comment]
I fixed it by copying the "static/dist/" folder from the master to development, and then by re-enabling the places and javascript api

**Comment by adbspurs** [comment]
@vihangm What is the Windows equiv of these instructions?

**Comment by wesleytsai** [comment]
I'm getting this on Windows:

Loading "sass.js" tasks...ERROR
>> Error: %1 is not a valid Win32 application.
>> \\?\C:\Users\tsaiw\bin\PokemonGo-Map\node_modules\grunt-sass\node_modules\node-sass\vendor\win32-x64-46\binding.node

**Comment by CtrlAltDefeat94** [comment]
@zanium how would I fix this to work with master?

**Comment by cstewart90** [comment]
@CtrlAltDefeat94 master has the dist folder so you don't need to do anything. If you mean the develop branch then you can reset to the commit before the dist folder was removed, since there has been no more commits yet.

    git checkout develop
    git pull
    git reset 0880e52 --hard

**Comment by CtrlAltDefeat94** [comment]
No, I am on master and it broke for me too. After I had done a git reset it worked, so not sure where it went wrong.

**Comment by marauder37** [comment]
I am so unimpressed with that commit

**Comment by micha279** [comment]
Same issue here, worked since yesterday flawlessly, but now even npm install fails O_o

Also suggested fix does not work:npm install
npm install grunt-cli
./node_modules/.bin/grunt build --> fails

Any help appreciated!

Thanks,
Michael

NPM Install Error Message:
sudo npm install
npm WARN prefer global node-gyp@3.4.0 should be installed with -g

> node-sass@3.8.0 install /home/micha279/PokemonGo-Map/PokemonGo-Map/node_modules/node-sass
> node scripts/install.js

sh: 1: node: not found
pogomap@0.1.0 /home/user/PokemonGo-Map/PokemonGo-Map
└── grunt-cli@1.2.0  extraneous

npm ERR! Linux 4.4.0-31-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! node-sass@3.8.0 install: `node scripts/install.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the node-sass@3.8.0 install script 'node scripts/install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs node-sass
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/user/PokemonGo-Map/PokemonGo-Map/npm-debug.log

**Comment by man0s** [comment]
All commands with sudo worked for me!

**Comment by tarthim** [comment]
Copying over /static/dist is what did it for me, after installing grunt/npm. Thanks @zanium

**Comment by zanium** [comment]
@CtrlAltDefeat94 just use the developer package instead of the master, and then transfer the static/dist folder over.
